### PR TITLE
Replaces Falling Falcons unit patches with Solar Devils unit patches.

### PR DESCRIPTION
**Pull request by blackdragonTOW**
Adds icons for and code for adding Solar Devils unit patches.

# Explain why it's good for the game

Were a different unit, so it wouldn't hurt to have our own patches.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
add: Added Solar Devil patches.
/:cl:

### CODE REVIEW

File: code/modules/clothing/head/helmet.dm [hunk 1]
```diff
@@ -322,7 +322,7 @@ GLOBAL_LIST_INIT(allowed_helmet_items, list(
 	/obj/item/device/flashlight/flare = "flare",
 	/obj/item/clothing/head/headset = "headset",
 	/obj/item/clothing/accessory/patch = "uscmpatch",
-	/obj/item/clothing/accessory/patch/falcon = "falconspatch",
+	/obj/item/clothing/accessory/patch/devils = "devilspatch",
```
Comment (private-tristan) [inline code/modules/clothing/head/helmet.dm:325]: should definitely be able to wear both, just add devilspatch as its' own entry instead of replacing it.

**Comment by blackdragonTOW** [comment]
Did you know that if you edit from an old version of a file, you sometimes dont pick up changes from previous verious? WHO KNEW?

**Comment by morrowwolf** [comment]
image

Looks like you accidentally deleted some sprites here.

**Comment by morrowwolf** [comment]
Also needs helmet overlay sprites

You can see other examples in: .icons/mob/humans/onmob/helmet_garb.dmi

**Pull request merged**